src/map/entities/battleentity.cpp

```diff
@@ -1188,6 +1188,41 @@ int16 CBattleEntity::getMod(Mod modID)
     return m_modStat[modID];
 }
 
+/************************************************************************
+ *                                                                      *
+ *  Get the highest value of the specified modifier across all gear     *
+ *                                                                      *
+ ************************************************************************/
+int16 CBattleEntity::getMaxGearMod(Mod modID)
```

Comment:
Now that this exists here, you can update the lua binding to use this implementation instead of having the same code exist twice

Reply:
Been a busy week, but I havent forgotten about this.

Reply:
Done, I think. Do tell if any other hing needs a change.